ode/log_setup.py

```diff
@@ -0,0 +1,74 @@
+import logging
+import sys
+from logging.handlers import RotatingFileHandler
+
+from PySide6.QtCore import QtMsgType, qInstallMessageHandler
+
+from ode import utils
+from ode.paths import LOGS_PATH
+
+
+def configure_logging():
+    """Configure logging for the application."""
+    # Create the logs directory if it doesn't exist
+    LOGS_PATH.mkdir(parents=True, exist_ok=True)
+
+    root_logger = logging.getLogger()
+    root_logger.setLevel(logging.INFO)
+    # Clear any existing handlers
+    for handler in root_logger.handlers[:]:
+        root_logger.removeHandler(handler)
```

Comment:
What other handlers could exist? In what scenario this happens?

I have bad experience with setup functions that start by cleaning leftovers that shouldn't exist in the first place.